### DISCUSSION TITLE
compinit:503: no such file or directory: /usr/local/share/zsh/site-functions/_brew_cask

### DISCUSSION THREAD

**Discussion by rogerl50** [ohmyzsh/ohmyzsh#10500] [Help]
When installing on MacOS (Catalina) I get the following error:
compinit:503: no such file or directory: /usr/local/share/zsh/site-functions/_brew_cask
even though that file clearly exists.

**Comment by mcornella** [comment]
[This has come up before](https://github.com/ohmyzsh/ohmyzsh/issues/9602#issuecomment-768663375). That file is a symlink to a file that doesn't exist.
